code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -254,6 +255,17 @@
 		. = TRUE
 	..()
 
+/datum/reagent/consumable/soymilk/overdose_process(mob/living/M, delta_time)
+	metabolization_rate = 1 * REAGENTS_METABOLISM
+	if(ishuman(M))
+		if(DT_PROB(5, delta_time))
+			var/static/list/filth = list("silicons", "ethereals", "plasmamen", "moths", "lizards") //Writing the OD speeches was fun, good times.
+			M.say(pick("Opening those soy bottles is hard, i think my strength is waning...", "Honestly, i think [pick (filth)] deserve rights.", "Did you see my profile? I got like 20 thousand likes!", "Did you see the NT announcement where they spoke out about violence against [pick(filth)]? I really should support them more!", "I wish we could get rid of security so there wouldn't be any more violence.", "This place needs more exposed brick walls, maybe even a barcade!"))
```

Comment:
change this to use pick_list_replacements
see: https://github.com/MisterBones2910/Merchan-TG-uild/tree/master/strings
`M.say(pick_list_replacements(FILE, "soy"), forced = /datum/reagent/consumable/soymilk)`

Reply:
First time working with that, i'll try.

Reply:
Alright, got it to work, neat.